src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,51 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import NotPeople from './NotPeople';
+import ErrorMessage from './ErrorMessage';
+import { Person } from '../types';
 
 export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(setPeople)
+      .catch(() => {
+        setIsError(true);
+      })
+      .finally(() => {
+        setIsLoading(false);
+      });
```

Comment:
Consider using async/await syntax for promises. It might improve readability of your code and make it more consistent.